deployment/ccip/changeset/v1_6/cs_enable_big_blocks.go

```diff
@@ -0,0 +1,284 @@
+
+	"github.com/ugorji/go/codec"
+
+	chain_selectors "github.com/smartcontractkit/chain-selectors"
+)
+
+var EnableBigBlockChangeset = cldf.CreateChangeSet(EnableBigBlocksLogic, EnableBigBlocksPreCondition)
+
+type EnableBigBlocksConfig struct {
+	APIURL   string
+	ChainSel uint64
+}
+
+// Payload to be sent in the HTTP POST request
+type RequestPayload struct {
+	Action    map[string]interface{} `json:"action"`    // Action details
+	Nonce     int64                  `json:"nonce"`     // Unique nonce for the request
+	Signature ECDSASignature         `json:"signature"` // ECDSA signature of the action
+}
+
+type ECDSASignature struct {
+	R string `json:"r"`
+	S string `json:"s"`
+	V byte   `json:"v"`
+}
+
+type Config struct {
+	URL               string        // RPC URL
+	ChainID           int64         // chain ID
+	VerifyingContract string        // Verifying contract address
+	RequestTimeout    time.Duration // HTTP request timeout
+}
+
+func EnableBigBlocksPreCondition(env cldf.Environment, cfg EnableBigBlocksConfig) error {
+	_, err := stateview.LoadOnchainState(env)
+	if err != nil {
+		return fmt.Errorf("failed to load onchain state: %w", err)
+	}
+	return nil
+}
+
+func EnableBigBlocksLogic(env cldf.Environment, cfg EnableBigBlocksConfig) (cldf.ChangesetOutput, error) {
+	chainIDStr, err := chain_selectors.GetChainIDFromSelector(cfg.ChainSel)
+
+	out := cldf.ChangesetOutput{}
+	if err != nil {
+		return out, fmt.Errorf("invalid chain id: %w", err)
+	}
+
+	if err != nil {
+		return out, fmt.Errorf("invalid private key: %w", err)
+	}
+
+	chainID, err := strconv.ParseUint(chainIDStr, 10, 64)
+	const maxInt64 = int64(^uint64(0) >> 1)
+	if chainID > uint64(maxInt64) {
+		return out, fmt.Errorf("chain ID too large for int64: %d", chainID)
+	}
+
+	if err != nil {
+		return out, fmt.Errorf("error converting string to uint64: %w", err)
+	}
+
+	action := map[string]interface{}{
+		"type":           "evmUserModify",
+		"usingBigBlocks": true,
+	}
+
+	chain, err := FindChainBySelector(env, cfg.ChainSel)
+
+	if err != nil {
+		return out, fmt.Errorf("error: %w finding chain by selector: %d", err, cfg.ChainSel)
+	}
+
+	// Verifying contract address for EIP-712 signing
+	defaultVerifyingContract := "0x0000000000000000000000000000000000000000"
+
+	// Default timeout for HTTP requests
+	defaultRequestTimeout := 10 * time.Second
+
+	nonce := time.Now().UnixMilli()
+	config := Config{
+		URL:               cfg.APIURL,
+		ChainID:           int64(chainID),
+		VerifyingContract: defaultVerifyingContract,
+		RequestTimeout:    defaultRequestTimeout,
+	}
+
+	sig, err := SignL1Action(action, nonce, true, config, chain)
+	if err != nil {
+		return out, fmt.Errorf("signing failed: %w", err)
+	}
+
+	err = sendRequest(RequestPayload{
+		Action:    action,
+		Nonce:     nonce,
+		Signature: sig,
+	}, config)
+	if err != nil {
+		return out, fmt.Errorf("send failed: %w", err)
+	}
+
+	out.Reports = append(out.Reports, operations.Report[any, any]{
```

Comment:
We can just return an empty output here, we don't need to include operations reports